docs/csv/example_files/precinct.txt

```diff
@@ -1 +1,4 @@
-id,ballot_style_id,electoral_district_ids,external_identifier_type,external_identifier_othertype,external_identifier_value,is_mail_only,locality_id,name,number,polling_location_ids,precinct_split_name,ward
\ No newline at end of file
+id,ballot_style_id,electoral_district_ids,external_identifier_type,external_identifier_othertype,external_identifier_value,is_mail_only,locality_id,name,number,polling_location_ids,precinct_split_name,ward
+pre90111,bs00010,"ed001",ocd-id,,"ocd-division/country:us",false,loc001,203 - GEORGETOWN,0203,"poll001 poll002",split13,5
+pre90112,bs00011,"ed002",fips,,42,false,loc001,203 - GEORGETOWN,0203,"poll003",split26,6
```

Comment:
The state of PA is a polling location for elections being held in Denver, CO? :)